data/sql_updates/large_aggregates.sql

```diff
@@ -217,19 +246,41 @@ select
 from combined
 ;
 
--- creates cumulative table per cycle from the data receipts in the large aggregates
-drop table if exists entity_receipts_chart;
-create table entity_receipts_chart as (select idx, type, month, year, cycle, adjusted_total_receipts, sum(adjusted_total_receipts) OVER (PARTITION BY cycle, type order by year, month, type desc) from large_aggregates_tmp);
+-- creates cumulative materialized view per cycle from the data receipts in the large aggregates
+drop materialized view if exists entity_receipts_chart;
+create materialized view entity_receipts_chart as (
```

Comment:
Should this also be named with `_mv_tmp` at the end to ensure that it is renamed and refreshed properly?